run.py

```diff
@@ -143,13 +143,27 @@ async def gateway_connect(self, data: Ready):
 
     async def gateway_player_update(self, data: Player):
         log.info(f"[System][Hutao Login Gateway] Update player {data}")
-        user_id = data.genshin.userid
-        cookie_token = data.genshin.cookie_token
+        
+        # Set variable data
+        user_id = data.discord.user_id
+        genshin = data.genshin
+
         # Update cookie_token
-        await self.db.execute("UPDATE user_accounts SET cookie_token = ? WHERE user_id = ?", (
-            user_id,
-            cookie_token
-        ))
+        _data = [
+            genshin.ltuid,
+            genshin.cookie_token
+        ]
+
+        # Set default value 
+        update_value = "ltuid = ?, cookie_token = ?"
+        # Check if ltoken is not empty string
+        if data.genshin.ltoken != "":
```

Comment:
In REST API it's can be to try send token again if user register has not successfully logged